### PR TITLE
[tflite] add profiling to label_image for tflite 

### PR DESCRIPTION
**Pull request by freedomtan**
use the profiling mechanism in dfae914b

build with something like:
```
bazel build --config android_arm64 \
  --cxxopt=-std=c++11 \
  --cxxopt=-DTFLITE_PROFILING_ENABLED \
  //tensorflow/contrib/lite/examples/label_image:label_image
```

run `label_image` will get something like:
```
./label_image -p 1
Loaded model ./mobilenet_quant_v1_224.tflite
resolved reporter
invoked
average time: 67.227 ms
    13.349, Node   0, OpCode   3, CONV_2D
     6.024, Node   1, OpCode   4, DEPTHWISE_CONV_2D
    11.847, Node   2, OpCode   3, CONV_2D
     3.927, Node   3, OpCode   4, DEPTHWISE_CONV_2D
     1.905, Node   4, OpCode   3, CONV_2D
     3.573, Node   5, OpCode   4, DEPTHWISE_CONV_2D
     2.344, Node   6, OpCode   3, CONV_2D
     0.964, Node   7, OpCode   4, DEPTHWISE_CONV_2D
     1.224, Node   8, OpCode   3, CONV_2D
     1.846, Node   9, OpCode   4, DEPTHWISE_CONV_2D
     2.181, Node  10, OpCode   3, CONV_2D
     0.454, Node  11, OpCode   4, DEPTHWISE_CONV_2D
     0.997, Node  12, OpCode   3, CONV_2D
     0.865, Node  13, OpCode   4, DEPTHWISE_CONV_2D
     1.844, Node  14, OpCode   3, CONV_2D
     0.753, Node  15, OpCode   4, DEPTHWISE_CONV_2D
     1.724, Node  16, OpCode   3, CONV_2D
     0.803, Node  17, OpCode   4, DEPTHWISE_CONV_2D
     1.698, Node  18, OpCode   3, CONV_2D
     0.794, Node  19, OpCode   4, DEPTHWISE_CONV_2D
     1.754, Node  20, OpCode   3, CONV_2D
     0.798, Node  21, OpCode   4, DEPTHWISE_CONV_2D
     1.704, Node  22, OpCode   3, CONV_2D
     0.204, Node  23, OpCode   4, DEPTHWISE_CONV_2D
     0.983, Node  24, OpCode   3, CONV_2D
     0.373, Node  25, OpCode   4, DEPTHWISE_CONV_2D
     1.791, Node  26, OpCode   3, CONV_2D
     0.067, Node  27, OpCode   1, AVERAGE_POOL_2D
     0.388, Node  28, OpCode   3, CONV_2D
     0.001, Node  29, OpCode  22, RESHAPE
     0.035, Node  30, OpCode  25, SOFTMAX
0.600: 458 bow tie
0.365: 653 military uniform
0.008: 835 suit
0.008: 611 jersey
0.004: 514 cornet
```

### CODE REVIEW

File: tensorflow/contrib/lite/examples/label_image/label_image.cc [hunk 1]
```diff
@@ -89,7 +107,7 @@ void RunInference(Settings* s) {
 
   tflite::ops::builtin::BuiltinOpResolver resolver;
 
-  tflite::InterpreterBuilder(*model, resolver)(&interpreter);
+  tflite::InterpreterBuilder (*model, resolver)(&interpreter);
```
Comment (andrehentz) [inline tensorflow/contrib/lite/examples/label_image/label_image.cc]: please remove extra whitespace

**Review by andrehentz — DISMISSED** [review]

**Comment by freedomtan** [comment]
Oops, sorry about that. Will remove it. That was actually generated by `clang-format -style=Google ...`

File: tensorflow/contrib/lite/examples/label_image/label_image.cc (hunk 1 above)
Comment (freedomtan) [reply tensorflow/contrib/lite/examples/label_image/label_image.cc]: done

**Review by andrehentz — APPROVED** [review]

**Comment by martinwicke** [comment]
Not sure this is an expected error for this change?

```
2018-04-26 14:22:17.797851: F tensorflow/python/lib/core/bfloat16.cc:664] Check failed: PyBfloat16_Type.tp_base != nullptr
```

**Comment by freedomtan** [comment]
@martinwicke: I didn't have such problem on Android devices. How to produce that?

**Pull request merged**